tools/yaml-templates/build-test-publish.yml

```diff
@@ -106,6 +106,14 @@ steps:
       flattenFolders: true
     displayName: 'Copy validDomains for CDN'
 
+  - task: CopyFiles@2
```

Comment:
It would be great to capture why this has the effect stated in a comment in the yaml file. The explanation you gave me makes sense, but I'm worried we will not remember how this works in a couple of months. The functionality described is a side effect of moving the file into a subfolder of the 7dttl folder.